.github/workflows/push.yml

```
@@ -0,0 +1,70 @@
+##############################################################################
+##############################################################################
+
+name: PUSH Workflow - All Branches
+
+on:
+  push:
+    branches:
+      - '**'   
+      
+env:
+    CODECOV_UNIQUE_NAME: CODECOV_UNIQUE_NAME-${{ github.run_id }}-${{ github.run_number }}
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 2 but found 4

(indentation)